pkgs/development/libraries/sundials/default.nix

```diff
@@ -0,0 +1,26 @@
+{ cmake, fetchurl, python, stdenv }:
+
+stdenv.mkDerivation rec {
+
+  pname = "sundials";
+  version = "3.1.0";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+  url = "https://computation.llnl.gov/projects/${pname}/download/${pname}-${version}.tar.gz";
+  sha256 = "0fnlrpj6qjamxahd70r4vsgv85kxbgh93gxqk5xzf9ln6a7jzm8q";
+  };
+
+  preConfigure = ''
+    export cmakeFlags="-DCMAKE_INSTALL_PREFIX=$out -DEXAMPLES_INSTALL_PATH=$out/share/examples $cmakeFlags"
```

Comment:
You can say `cmakeFlags = ...` as a top-level attribute rather than setting it in preConfigure I believe.

Reply:
Yes, that should be possible. See [here](https://github.com/NixOS/nixpkgs/blob/105255e6f5c6854ee9629ff1274f19711806399a/pkgs/applications/science/robotics/yarp/default.nix#L18-L22) for an example.